src/types/index.ts

```diff
@@ -26,12 +26,11 @@ export interface WalletConnection {
   l2Signer: L2Signer;
 }
 
-export type EthNetwork = 'dev' | 'ropsten' | 'mainnet';
-
 export interface Config {
   api: Configuration;
   starkContractAddress: string;
```

Comment:
This should be `verifierAddress` or `coreAddress` rather than `starkContractAddress`.

Reply:
Will we remove `registrationContractAddress` at any point in the near future (e.g. when upgrading to StarkEx v4)? @igx-C

Reply:
In v4.0 - 
- Deposits and withdrawals will no longer require registration, saving users $
- Registration is still required for any unhappy (uncommon) flow: escape request, deposit-cancellation request. Note, the new `registerEthAddress` registration function is costly and shouldn’t normally be used, so I imagine we can remove this. I don't see much value in having a wrapper contract for these flows.

Reply:
starkContractAddress renamed to coreContractAddress
[0c1bfc6](https://github.com/immutable/imx-core-sdk/pull/121/commits/0c1bfc686524185b4e22fc3bac5cf29f0b63a34a)